lib/rust/ensogl/component/text/src/component/area.rs

```diff
@@ -1,6 +1,7 @@
 //! The text area implementation. It serves the purpose of single and multi-line text labels and
 //! text editors.
 
+
```

Comment:
This file with hundreds of pragmas is ugly as hell. But Im ok with leaving it this way for now. I can try fixing it when implementing widgets (which require big changes to this file)